docs/_v9_1/1_setup.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Initial setup
+identification: setup
+layout: default
+---
+
+The client library is available as a nuget package. The client library (and associated nuget package) is updated regularly as new functionality is added. 
+
+
+To install the nuget package, follow these steps in Visual Studio/Rider:
+
+1. Select _TOOLS -> nuget Package Manager -> Manage nuget Packages Solution..._
+
+1. Search for _Digipost.Api.Client. Multiple packages will appear. Install those necessary for you. Make sure you _DON'T_ install the _digipost-api-client_ packages. Those are .NET Framework libraries with an unfortunately similar name.
```

Comment:
Kanskje vi bør skrive om denne til heller å forklare at denne dokumentasjonen er for .NET core, og lenke til versjon 8 for de som bruker .NET framework? Men likevel viktig å forklare at det er forskjellige navn?

Reply:
Fikset i 35bf7a5